cutscene_engine/data/quest_giver_conversation.json

```diff
@@ -0,0 +1,5 @@
+{
+    "001" : {"name":"Godette", "expression": "neutral", "text":"We are adventurers, right?." },
+    "002" : {"name":"Robi", "expression": "neutral", "text":"I guess so, why?" },
+    "003" : {"name":"Godette", "expression": "neutral", "text":"Let's go on a quest!" }
+}
```

Comment:
I'm iffy about this format. Rather than using the `"001"`, `"002"`, etc keys, I think it would be better to store the dialogue in an array. It could be really frustrating to make changes to this. If designers are aware and ok with this then fine.

Reply:
I can see why, not sure how to do that. This whole system was based heavily off a pre-existing, open dialogue system from GDquest. This was the format that they had for their open source open RPG project that I based the current system off. As always I can talk with Julian about this

Reply:
I'd just recommend using a more human readable value as a key. For example, "godette".